client/src/App.tsx

```diff
@@ -14,7 +13,7 @@ if (hostname === "localhost" || hostname === "127.0.0.1" || port === "3000") {
   axios.defaults.baseURL = `//${hostname}:8000`;
   console.log("Running on Localhost", axios.defaults.baseURL);
 } else {
-  axios.defaults.baseURL = `https://api.${hostname}`;
+  axios.defaults.baseURL = `http://api.${hostname}`;
```

Comment:
Not sure if this is necessary, @cloudyyoung would know better

Reply:
Don't think this is necessary for now. Backend is expected to be hosted with https.